wellregistry/registry/admin/monitoring_location.py

```diff
@@ -15,13 +15,197 @@
 from .fetch_from_nwis import FetchFromNwisView
 
 USGS_AGENCY_CD = 'USGS'
-
+tooltips = [{
+		'form_field': 'display_flag',
+		'tooltip': 'Flag that controls whether the site will be displayed on the NGWMN Data Portal (Yes/No)'
+	},
+	{
+		'form_field': 'agency',
+		'tooltip': 'The name of the agency or organization that collects, owns andor manages the data. \
+ If you don\'t see your agency or organization in the list contact Jessica Lucido(lucido @usgs.gov).'
```

Comment:
This contact should also be changed to Candice.

Reply:
fixed